src/components/ui/TimeSlotParticipantsPanel.tsx

```diff
@@ -15,7 +17,7 @@ const AvailablePerson: React.FC<{
     >
       <div className="font-semibold">{name}</div>
       <IoEarthSharp className="text-md ml-2 text-gray-500" />
-      <div className="text-gray-500">Chicago (GMT-8)</div>
+      <div className="text-gray-500">{formatTimeZoneTag(timeZoneTag)}</div>
     </div>
   );
```

Comment:
so we are using `formatTimeZoneTag(timeZoneTag)` on display but saving the raw `timeZone`? If so then I'd say naming the field `timeZoneTag` seems a bit weird cuz we are not acc storing `tags` but stuff that gets converted to tags. Perhaps a better way of naming it might be `timeZoneValue` or sth

Reply:
We are saving strings in the format of `timeZoneTag` in the field `timeZone` in the tables `Event` and `Participate` in the database. Hence, the code you mentioned here is handling a correct `timeZoneTag` value.

<img width="196" alt="Screenshot 2023-06-25 at 11 00 49 PM" src="https://github.com/ParachuteTeam/Parachute/assets/30245379/aeaa8110-8aba-4f2d-b8de-e01cfb7604b7">

This is because we already have the field `timeZone` in the `Participate` table when we planned to add a time zone field in the `Event` table, so I simply used the existing name `timeZone`, since they will be storing values in the same format.

Also, I think the word `timeZone` is general so that we don't have to refactor the database structure again if we want to change the design. So far, this convention is used in only these two places and nowhere else, so I think it won't confuse us too much.

Just here for reference: `timeZoneTag` is a concept I introduced in a previous PR: time zone location string (Asia/Shanghai) and time zone offset string (GMT+8) separated by a comma so that the result will be "Asia/Shanghai,GMT+8". All functions in utils.ts depend on this format. `formatTimeZoneTag` converts "Asia/Shanghai,GMT+8" to "Asia/Shanghai (GMT+8)" for better display.